src/corner_kick/src/containers/Canvas/utils/draw.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * This file specifies functions to draw various types of shapes
+ */
+
+import {
+    IArcShape,
+    IEllipseShape,
+    ILineShape,
+    IPolyShape,
+    IRectShape,
+    IShape,
+} from 'SRC/types';
+
+/**
+ * Sets the correct draw style (fill and stroke). All sprites are white to ensure
+ * correct tinting accuracy
```

Comment:
What if we want to draw a shape that's not white? Do we just apply the color via a shader after we grab the texture from the spritesheet? If so, we should make a quick note to that effect here.

Reply:
Correct, shape has to be white in order for tinting to work properly.

Reply:
Added a more explicit comment
```
/**
 * Sets the correct draw style (fill and stroke). All sprites are white to ensure
 * correct tinting accuracy. Actual tinting is specified when creating the sprite.
 * @param ctx an canvas rendering context
 * @param shape the shape description object
 */
```